Map layers control and styles

dhakelila opened the pull request:
This PR:
- gets the layers from the API and draws it on the map within its category. 
-Add layers control to add/remove layers within categories.
-Improves map styles. 
-Fit map bounds to the current layers.

simaob commented:
- Not sure about the fixing the map bounds to the layer. It becomes a bit jumpy, so not sure if it adds a lot of value.
- Change label of "To Manage" to. "Tasking Manager tasks"
- Zoom controls seem to be the wrong way around? If I click the minus it zooms in instead of out...
- styles of the map in the backend are affected by these changes and controls are no longer visible, check with @edbrett to fix it.

@juancarlosalonso: could we try a different colour for the tasking manager polygons. the white seems a bit pale and gets mixed up with the borders of the basemap.

dhakelila commented:
I fixed those. 
Now, 
-Zoom icons are in the correct order. 
-Changed label for switcher.
-Styles are only applying to front side. 
-I remove the bounding function for now but I'll manage it to fit bounds only the first time you get the map, no when switching on/off. I think it would be useful in the case you are seeing, for example, task how are only placed in Africa or wherever.

dhakelila commented:
I forgot to mention. I talk with Juancarlos about polygons color and we set the white more opaque. 
Let's see if that works for you. Anyway, I didn't see the white as transparent as you show in your pic.
